main/getting-started/intro-zoe.md

```diff
@@ -1,358 +1,286 @@
-including concepts not covered here. 
+## What is Zoe?
 
-You can run different types of smart contracts on Zoe. In this document, 
-for the most part we focus on Agoric’s Atomic Swap demo contract, but 
-sometimes refer to how Zoe works with other contract types.
+Zoe is Agoric's smart contract framework. You should use Zoe to:
 
-## What is Zoe?
+* **Run your code on-chain**
+* **Mint new digital assets**
+* **Credibly trade assets**
+
+Zoe relies on [ERTP](./ertp-introduction.md), our token standard for fungible and non-fungible
+tokens. 
+
+## Why should I use Zoe?
+
+### For Users ###
+
+**Zoe is safer.** Putting digital assets in a smart contract has
+traditionally meant you might lose them. But Zoe guarantees that you
+will either get what you wanted or get a full refund. You will never
+leave a contract empty-handed, even if the smart contract is buggy or
+malicious. 
+
+### For Developers ###
+
+**Zoe is easier.**  Writing a smart contract has traditionally meant
+learning a completely new, untried language. And let's hope you don't
+make any mistakes - if you do, your users might lose millions.
+
+Zoe smart contracts, on the other hand, are written in a secure subset
+of JavaScript. Moreover, Zoe handles all of the escrowing of digital
+assets provided by your users and the subsequent payout of digital
+assets to your users. **Your contract, even if it's buggy, can't cause
+your users to lose their assets.**
+
+## How do I use Zoe?
+
+There are three ways you can get started using Zoe:
+
+* you can write a new contract and install it on Zoe
+* you can create a new `instance` of a contract already installed on
+  Zoe
+* you can receive an `invitation` to join a contract
+  instance that someone else has started.
+
+Agoric has written [a number of example contracts
+that you can use](/zoe/guide/contracts/), including:
+* a [Uniswap
+implementation](/zoe/guide/contracts/multipoolAutoswap.md)
+* a [covered
+call option contract](/zoe/guide/contracts/covered-call.md)
+* an [OTC
+Desk market maker
+contract](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/otcDesk.js)
+* contracts for [minting fungible](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/mintPayments.js) and [non-fungible tokens](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/mintAndSellNFT.js)
+
+## Using a Zoe smart contract: an example
+
+Let's imagine that our friend Alice has sent a `Zoe invitation` to
+[our wallet](/wallet-api.md). This invitation will let us join Alice's
+running contract instance in a particular role. Without an invitation,
+we are not able to join.
+
+Compare this to a smart contract on Ethereum. On Ethereum, the smart
+contract developer must guard against malicious calls and store an
+internal access control list to check whether the message sender is
+allowed to send such a message. Zoe, built on Agoric's [object
```

Comment:
Hmm, why? `can send` sounds like whether it is mechanically possible, which gives the wrong impression, since it's more like `may send`. I think `allowed` is correct.